Application/Frontend/__tests__/FriendsTab.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { MongoClient } from 'mongodb';
+import handler from '@/pages/api/FriendsTab'; 
+import { NextApiRequest, NextApiResponse } from 'next';
+
+// Mock MongoClient and dbCollection
+jest.mock('mongodb', () => ({
+  MongoClient: jest.fn()
+}));
+
+const mockDbCollection = jest.fn();
+const mockClose = jest.fn();
+
+describe('handler function', () => {
+  let req: NextApiRequest;
+  let res: NextApiResponse;
+  
+  beforeEach(() => {
+    req = {
+      method: 'POST',
```

Comment:
It would be better to use a `GET` request here as you are retrieving data. In addition, you are not transmitting sensitive data, so I would change this to a `GET` request.